src/data/configs/no_math_code_20bn.yaml

```diff
@@ -0,0 +1,38 @@
+sources:
+  - name: cc_en_middle
+    source_coefficient: 1
+  - name: falcon
+    source_coefficient: 1
+  - name: cc_en_tail
+    source_coefficient: 1
+  - name: cc_en_head
+    source_coefficient: 1
+  - name: c4
+    source_coefficient: 1
+  - name: starcoder
+    source_coefficient: 0  # Remove code
+  - name: reddit
+    source_coefficient: 1
+  - name: pes2o
+    source_coefficient: 1
+  - name: arxiv
+    source_coefficient: 1
+  - name: wiki
+    source_coefficient: 1
+  - name: tulu_flan
+    source_coefficient: 1
+  - name: stackexchange
+    source_coefficient: 0  # TODO Does this count in math / code bucket
```

Comment:
I'm ambivalent, either way.

Reply:
> I'm ambivalent, either way.

Will keep as `0` and remove the comment - shouldn't make too much difference